src/ensembl/src/content/app/browser/track-panel/track-panel-modal/modal-views/TrackPanelBookmarks.tsx

```diff
@@ -137,25 +104,16 @@ export const TrackPanelBookmarks = () => {
   };
 
   return (
-    <section className="trackPanelBookmarks">
-      <div className={styles.title}>Bookmarks</div>
-      {exampleEnsObjects.length ? <ExampleLinks /> : null}
-      {limitedPreviouslyViewedObjects.length ? (
+    <section className={styles.trackPanelBookmarks}>
+      <div className={styles.title}>Previously viewed</div>
+      {previouslyViewedObjects.length ? (
         <>
-          <div data-test-id="previously viewed" className={styles.sectionTitle}>
-            Previously viewed
-            {previouslyViewedObjects.length > 20 && (
-              <span className={styles.ellipsis}>
-                <ImageButton
-                  status={Status.DEFAULT}
-                  description={'View all'}
-                  image={EllipsisIcon}
-                  onClick={onEllipsisClick}
-                />
-              </span>
-            )}
-          </div>
           <PreviouslyViewedLinks />
+          {previouslyViewedObjects.length > 20 && (
+            <div className={styles.more} onClick={onMoreClick}>
+              more...
+            </div>
```

Comment:
This whole line, rather than just the word "more..." becomes clickable:

<img src="https://user-images.githubusercontent.com/6834224/124113474-64a3b580-da63-11eb-92e4-91380653cf8d.png" width="350px">

<img src="https://user-images.githubusercontent.com/6834224/124113709-a5033380-da63-11eb-8650-3f78199f20af.png" width="350px">


I think you want a span inside of the div; and the styles and click handler should be put on that span: